### PR TITLE
New TRG 1.07 - End User Manual

### PR DESCRIPTION
**Pull request by Siegfriedk**
Please be aware: while I created this PR, this TRG is not from me, i'm only preparing and moderating the merging of the new TRG 1.07.

### CODE REVIEW

File: docs/release/trg-1/trg-1-07.md
Comment (SebastianBezold) [inline docs/release/trg-1/trg-1-07.md:1]: In Why:
  - "Typical contents are screenshots" -> I think this leads to docs mainly consisting of just screenshots. I would emphasize the use case __descriptions__ and only add screenshot as good practice, where needed.

In Description:
- The second sentence is weird. I think it can be deleted without replacement

In Best Practice
- Same feedback as for the other TRG proposals...
- Do not invent phrases like "up-revision".
- Remove the "hand over" part. 
- appropriate maturity is completely unspecific
- Last bullet point `README.md` instead of readme.md

**Review by SebastianBezold — CHANGES_REQUESTED** [review]

File: docs/release/trg-1/trg-1-07.md
Comment (carslen) [inline docs/release/trg-1/trg-1-07.md:1]: Comment from previuos PR #818 


Struggeling with this one. Who is an end-user? An user using the Portal? Not all Catena-X/Tractus-X components will be used by end-users (e.g. EDC).

I'd also suggest to rename to User Manual as it's a more common term for documentation. End-User Manual reminds me to a formal thing like End-User-Agreement.

**Review by carslen — DISMISSED** [review]

**Comment by SebastianBezold** [comment]
Fine with the latest changes, but as with the other TRGs, I would like to discuss it in the committer meeting before merging

File: docs/release/trg-1/trg-1-07.md
Comment (Siegfriedk) [reply docs/release/trg-1/trg-1-07.md:1]: End-User is meant to separate between a user like ops corp etc. and the real end user

Comment (Siegfriedk) [reply docs/release/trg-1/trg-1-07.md:1]: Why -> Screenshot adjusted
Description -> second sentence is gone i think
Best Practice -> up-revision is gone, hand over is gone, majurity adjusted, README.md fixed

**Review by florianrusch-zf — APPROVED** [review]

**Review by FaGru3n — DISMISSED** [review]
LGTM

**Review by evegufy — DISMISSED** [review]

File: docs/release/trg-1/trg-1-07.md
Comment (mhellmeier) [inline docs/release/trg-1/trg-1-07.md:13]: In the other TRGs, like TRG 1.06 (see #825), this section always indicates if the section (in this case an end user manual) **must** be provided or **shall** be provided. I would also vote for adding this information here.

**Review by mhellmeier — CHANGES_REQUESTED** [review]

File: docs/release/trg-1/trg-1-07.md
Comment (Siegfriedk) [reply docs/release/trg-1/trg-1-07.md:13]: This is part of the ## Description. Your referenced TRG doesn't say it has to exist or not, it only says it shall help operators.

I'm fine with aligning it across TRGs, but for the sake of getting this merged before my capacity runs out: Pls approve and lets discuss alignment of it ater

**Review by SebastianBezold — DISMISSED** [review]

File: docs/release/trg-1/trg-1-07.md
Comment (mhellmeier) [reply docs/release/trg-1/trg-1-07.md:13]: But what is the correct answer to my question: Does this section relate to a _must_ or to a _shall_?

Comment (Siegfriedk) [reply docs/release/trg-1/trg-1-07.md:13]: It is a must. It is a must if there is an enduser involved.

**Review by almadigabor — APPROVED** [review]

File: docs/release/trg-1/trg-1-07.md
Comment (mhellmeier) [reply docs/release/trg-1/trg-1-07.md:13]: Ok, then this should be adjusted in the text.

Comment (Siegfriedk) [reply docs/release/trg-1/trg-1-07.md:13]: Its updated. Pls check

File: docs/release/trg-1/trg-1-07.md
Comment (tom-rm-meyer-ISST) [inline docs/release/trg-1/trg-1-07.md:27]: I would add also two practices: role specifics and if you have an ui, make it available there (should be close to styleguide which has not yet been released)

```suggestion
- Anchor a link to your end user manual within your `README.md` (see [TRG 1.01](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-1))
- Within the documentation consider role specifics of end users for better understanding
- If your application has an UI, make the documentation available within the document
```

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
overall fine from my side, I would add two best practices

File: docs/release/trg-1/trg-1-07.md
Comment (mhellmeier) [reply docs/release/trg-1/trg-1-07.md:13]: Great, thank you!

**Review by mhellmeier — DISMISSED** [review]

**Review by matbmoser — APPROVED** [review]
We have adopted this also, good idea, approved from my side

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM!

**Review by evegufy — APPROVED** [review]

**Review by mhellmeier — COMMENTED** [review]
LGTM

**Pull request merged**
